voxel51/platform/api.py

```diff
@@ -136,6 +195,26 @@ def upload_job_output_as_data(self, job_id, path):
         _validate_response(res)
         return _parse_json_response(res)["data"]["data_id"]
 
+    def _get_job_url(self, task_config, url_type):
+        '''Retrieves a signed URL to post job information or output.
+
+        Args:
+            task_config (voxel51.platform.task.TaskConfig): the task config
+            url_type (str): one of "status", "log", or "output"
+
+        Returns:
+            a RemotePathConfig object
+        '''
+        endpoint = (self.base_url + "/jobs/" + task_config.job_id +
+                    "/url/" + url_type)
+        res = self._requests.get(endpoint, headers=self._header)
+        try:
+            _validate_response(res)
+            return voxu.RemotePathConfig(_parse_json_response(res))
+        except (APIError, HTTPError) as e:
+            logger.warning(e)
+            return getattr(task_config, url_type)
```

Comment:
Add an additional warning message saying that we're defaulting to the pre-populated URLs?